src/cli.rs

```diff
@@ -211,6 +212,13 @@ pub struct RootOpts {
     /// `--watch-config`.
     #[arg(long, env = "VECTOR_ALLOW_EMPTY_CONFIG", default_value = "false")]
     pub allow_empty_config: bool,
+
+    /// Turn on strict mode for environment variable interpolation. When set, interpolation of a
+    /// missing environment variable in configuration files will cause an error instead of a
+    /// warning, which will result in a failure to load any such configuration file. This defaults
+    /// to false, but that default is deprecated and will be changed to strict in future versions.
+    #[arg(long, env = "VECTOR_STRICT_ENV_VARS", default_value = "false")]
+    pub strict_env_vars: bool,
```

Comment:
I think this is missing from the manually maintained `website/cue/reference/cli.cue`

Reply:
Fixed in c9abff954